src/status_im/notifications/core.cljs

```diff
@@ -144,6 +148,11 @@
      ;; TODO: Rehydrate message id
      :id   id})
 
+  (defn- get-contact-name [{:keys [db] :as cofx} from]
+    (if (accounts.db/logged-in? cofx)
```

Comment:
Right now the `to` key/value pair tells us who the 1:1 message is directed to. In the future when we start supporting mentions, we should add a new key/value pair to inform in which channel we were mentioned in. You do raise a valid concern since we should be checking in `show-notification?` that the current account is actually the target account (assuming we don't want to be showing notifications for non-signed-in accounts).